test/load-config.js

```diff
@@ -0,0 +1,13 @@
+var fs = require('fs')
+
+var config = {}
+if (fs.existsSync('./test/config.json')) {
+  config = JSON.parse(fs.readFileSync('./test/config.json'))
+  for (var k in config) {
+    if (config.hasOwnProperty(k)) {
+      process.env[k] = config[k];
```

Comment:
This feels a bit dicey - I think it's best to pass variables to actual SDK code via env, but test code reads configs from the object returned below. Let me know what you think.